daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/unparsers/UState.scala

```diff
@@ -386,7 +386,7 @@ final class UStateForSuspension(
   override def advanceAccessor: InfosetAccessor = die
   override def inspect: Boolean = die
   override def inspectAccessor: InfosetAccessor = die
-  override def fini: Unit = {
+  override def fini(): Unit = {
```

Comment:
This finalize method not being called suggests it is not needed. Could be dead code.

Reply:
Not dead.